scripts/ThermalExpansion.zs

```diff
@@ -186,4 +187,32 @@ print("--- loading ThermalExpansion.zs ---");
 # Hardened Cell Frame
 	mods.actuallyadditions.Empowerer.addRecipe(<thermalexpansion:frame:129>, <thermalexpansion:frame:128>, <immersiveengineering:material:2>, <thermalfoundation:material:354>, <thermalfoundation:material:290>, <mekanism:basicblock:8>, 2500000, 200, [0.25, 0.25, 0.29]);
 
+# Remove creative kit crafts (no actual creative conversion kit)
+recipes.removeByRecipeName("thermalexpansion:cell_17");
+recipes.removeByRecipeName("thermalexpansion:cell_16");
+recipes.removeByRecipeName("thermalexpansion:cell_15");
+recipes.removeByRecipeName("thermalexpansion:cell_14");
+recipes.removeByRecipeName("thermalexpansion:cell_18");
+recipes.removeByRecipeName("thermalexpansion:tank_18");
+recipes.removeByRecipeName("thermalexpansion:tank_16");
+recipes.removeByRecipeName("thermalexpansion:tank_17");
+recipes.removeByRecipeName("thermalexpansion:tank_14");
+recipes.removeByRecipeName("thermalexpansion:tank_15");
+
+// # Dynamo harder recipes
+function remakeDynamo(name as string, item as IItemStack, i1 as IIngredient, i2 as IIngredient){
```

Comment:
Please undo the recipe changes here, at least for the time being. I know I've previously said I liked them, but I agree with the point that the recipes should not be made harder while they remain largely unused.